python/lsst/meas/astrom/matchOptimisticB.py

```diff
@@ -262,6 +255,19 @@ def matchObjectsToSources(self, refCat, sourceCat, wcs, refFluxField, maxMatchDi
             usableSourceCat=usableSourceCat,
         )
 
+    def _getIsGoodKeys(self, schema):
+        self.edgeKey = schema["base_PixelFlags_flag_edge"].asKey()
+        self.interpolatedCenterKey = schema["base_PixelFlags_flag_interpolatedCenter"].asKey()
+        self.saturatedKey = schema["base_PixelFlags_flag_saturated"].asKey()
+
+    def _isGoodTest(self, source):
+        # This is a hard coded version of the isGood flag from the old SourceInfo class that used to be
```

Comment:
I've changed this text to a docstring. We can discuss what changes we can make to BaseSourceSelector to make it play nice with match-lists and other "catalog like" objects next week if you would like.